src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -126,6 +156,30 @@ Specific iEEG fields MUST be present:
 
 Specific iEEG fields SHOULD be present:
 
+{{ MACROS___make_metadata_table(
+   {
+      "DCOffsetCorrection": "RECOMMENDED",
```

Comment:
this field was deprecated in https://github.com/bids-standard/bids-specification/pull/799 - sorry to just notice now (should have tagged you)

Reply:
Good catch. Thanks!

```suggestion
      "DCOffsetCorrection": "DEPRECATED",
```